### DISCUSSION TITLE
Can we build Onnxruntime nuget package with multiple EP at same time?

### DISCUSSION THREAD

**Discussion by DizzyZff** [microsoft/onnxruntime#20642] [EP Q&A]
I am currently working on a project that involves integrating the ONNX Runtime into a C# application, and I need to support multiple execution providers (EPs) such as CUDA, DirectML (DML), and CPU. My aim is to package these configurations into a single NuGet package to simplify deployment and management.

I understand that ONNX Runtime supports various EPs and can be built with support for multiple EPs. However, I am unsure how to configure the build process to include multiple EPs in a single NuGet package effectively.

This is the command I use:

`.\build.bat --use_cuda --cudnn_home "C:\Program Files\NVIDIA GPU Computing Toolkit\CUDNN\8.x" --cuda_home "C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v12.2" --use_dml --use_openvino CPU --build_shared_lib --build_nuget --config RelWithDebInfo`

But it seems to only build openvino package for me.

**Comment by emako** [comment]
Hoping for the return of openvino packages
